### ISSUE TITLE
Make it possible to pass `io.Writer` to `Migrator.Status`

### ISSUE THREAD

**Issue by aeneasr** [gobuffalo/pop#473]
## Description

At the moment, `pop.Migrator` exposes `Status()` which writes directly to `os.Stdout`:

https://github.com/gobuffalo/pop/blob/master/migrator.go#L200

### Expected Behavior

I would expect to be able to pass my own `io.Writer` there, because I want to, for example, redirect the output to a logger or somewhere else.

This could easily be solvable with:

```go
type Migrator struct {
	Connection *Connection
	SchemaPath string
	Migrations map[string]Migrations
+      Stdout io.Writer  
}
```

Happy to contribute a PR.

**Comment by stanislas-m** [comment]
Fixed with #498.

**Comment by aeneasr** [comment]
Awesome!
